README.md

```diff
@@ -20,7 +20,7 @@ and metrics from your application. You can analyze them using [Prometheus], [Jae
 use opentelemetry::{api::TracerGenerics, global, sdk};
 
 fn main() {
-    global::set_provider(sdk::Provider::default());
+    global::set_tracer_provider(sdk::Provider::default());
```

Comment:
I think we need to hold the guard here, otherwise this example will not work as expected.
```suggestion
    let _guard = global::set_tracer_provider(sdk::Provider::default());
```